v3/deployment.go

```diff
@@ -134,42 +116,20 @@ var _ = V3Describe("deployment", func() {
 
 				Eventually(func(g Gomega) {
 					session := cf.Cf("app", appName).Wait()
-					// previous deployment
-					g.Expect(session).Should(Say(`#0\s+running`))
-					g.Expect(session).Should(Say(`#1\s+running`))
-					// new deployment
-					g.Expect(session).Should(Say(`#0\s+starting`))
-					g.Expect(session).Should(Say(`#1\s+starting`))
-					g.Expect(session).ShouldNot(Say(`#2\s+starting`))
 					g.Expect(session).Should(Say("Active deployment with status DEPLOYING"))
 					g.Expect(session).Should(Say(`strategy:\s+rolling`))
 					g.Expect(session).Should(Say(`max-in-flight:\s+2`))
-				})
+				}).Should((Succeed()))
```

Comment:
It might technically work, but your succeed has an extra set of parens around it. Can you please remove them and check the other succeeds as well?

Thanks for making this change!